pkg/proto/nbcontract/v1/tlsbootstrappingconfig.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package nbcontract.v1;
+import "pkg/proto/nbcontract/v1/featurestate.proto";
+
+message TLSBootstrappingConfig {
+    // discuss whether this should be featureState or bool
+    // understand how defaulting works for enum, because we want default to be false
+    FeatureState tls_bootstrapping_status = 1;
+    FeatureState secure_tls_bootstrapping_status = 2;
```

Comment:
Would something like bootstrapping mode enum make more sense? These are mutually exclusive, aren't they?

Reply:
Both variables are in the cse_cmd.sh, so I put them as is.
I did a quick search and find that there is a condition in cse_config.sh that
`if [ "${ENABLE_SECURE_TLS_BOOTSTRAPPING}" == "true" ] || [ "${ENABLE_TLS_BOOTSTRAPPING}" == "true" ]; then`
So it seems they are not mutually exclusive.
I left a comment to follow up with this in the next milestone when we are extracting the logics. We will further follow up with the feature owner to understand more.

Reply:
> Both variables are in the cse_cmd.sh, so I put them as is.

That should not be a constraint, you can generate cse_cmd.sh correctly even if Go data structure is shaped differently (e.g. using an expression in the template). But keeping both closely aligned may be ok at this state - if we plan to revisit it later.

> So it seems they are not mutually exclusive.

I am fairly certain they are - in a sense they will never be both true at the same time. There are things that need to happen in either case, thus the logic in the script.

Reply:
Okay. Will review the TLS things together in this coming milestone.